symplyphysics/laws/kinematic/damped_oscillations/exponential_decay_constant.py

```diff
@@ -0,0 +1,47 @@
+from sympy import Eq
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    angle_type,
+    dimensionless,
+    validate_input,
+    validate_output,
+    print_expression,
+)
+
+# Description
+## For any (positive) value of the damping ratio, the position of a damped oscillator
+## is subject to exponential decay, i.e. its position is proportional to exp(-lambda*t)
+## where lambda is the exponential decay constant and t is time.
+
+# Law: lambda = omega * zeta
```

Comment:
Maybe it is better to write it in form `zeta = lambda / omega`. I think it is more usual to know decay rate and calculate damping ratio from it.

Reply:
I'll look into it

Reply:
I guess you're right, it makes more sense practically